sdks/common/fn-api/src/test/resources/org/apache/beam/fn/v1/standard_coders.yaml

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
+1 to this. I think some comments describing the scope of the test data here would probably have forestalled my lack of understanding of the intent.